### PR TITLE
Use `model_name` instead of inferring it from class name

### PR DESCRIPTION
**Pull request by ersatzryan**
This allows the use of decorator pattern in views.

### CODE REVIEW

File: lib/x-editable-rails/view_helpers.rb [hunk 1]
```diff
@@ -33,7 +33,7 @@ def editable(object, method, options = {})
           html_options = options.delete(:html){ Hash.new }
 
           if xeditable?(object)
-            model   = object.class.name.split('::').last.underscore
+            model   = object.model_name.to_s.underscore
```
Comment (laserlemon) [inline lib/x-editable-rails/view_helpers.rb]: If this spacing were fixed, could this be merged?

Comment (jirikolarik) [inline lib/x-editable-rails/view_helpers.rb]: It can be merged either way

Comment (laserlemon) [inline lib/x-editable-rails/view_helpers.rb]: Great, thank you!

**Comment by laserlemon** [comment]
Any chance a new patch version could be cut? Thanks again!

**Comment by jirikolarik** [comment]
@laserlemon bump it and I make release :)

**Comment by laserlemon** [comment]
https://github.com/werein/x-editable-rails/pull/56

**Comment by jirikolarik** [comment]
Here you go

**Comment by laserlemon** [comment]
Thanks!

**Pull request merged**